src/interpreter/ByteCode.h

```diff
@@ -539,25 +593,43 @@ class InitializeClass : public ByteCode {
             String* m_name;
         }; // CreateClass
         struct {
-            size_t m_fieldSize;
-            size_t m_staticFieldSize;
+            size_t m_fieldSize : 16;
```

Comment:
How about use ```uint16_t``` instead of ```size_t : 16```?
or
I think you should use ```__attribute__((packed))```

Reply:
That's good idea!